config/default.yml

```diff
@@ -156,6 +156,10 @@ RSpec/ExampleWording:
   CustomTransform:
     have: has
 
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecfilepath
+RSpec/FilePath:
+  Enabled: false # It's deprecated anyway
```

Comment:
I removed this deprecated cop in the previous PR, but it is still enabled by default.